src/imgui/ImGuiUtils.hh

```diff
@@ -63,6 +63,7 @@ inline void RightAlignText(std::string_view text, std::string_view maxWidthText)
 namespace openmsx {
 
 class BooleanSetting;
+class Debuggable;
```

Comment:
Detail: this forward declaration is not needed (now that we use a `function_ref` rather than a `Debuggable`).